pkg/ee/kubelb/controller.go

```diff
@@ -142,6 +144,16 @@ func (r *reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		return reconcile.Result{}, nil
 	}
 
+	if cluster.Status.NamespaceName == "" {
+		r.log.Debug("Skipping cluster reconciling because it has no namespace yet")
+		return reconcile.Result{RequeueAfter: healthCheckPeriod}, nil
+	}
+
+	if cluster.Status.ExtendedHealth.Apiserver != kubermaticv1.HealthStatusUp {
+		r.log.Debug("API server is not running, trying again in 5 seconds")
```

Comment:
`r.log.Debugf("API server is not running, trying again in %v", healthCheckPeriod)` to prevent code drift

Reply:
Updated